.github/workflows/lint-test.yaml

```diff
@@ -58,32 +47,7 @@ jobs:
 
       - uses: ./.github/actions/prepare-k8s
 
-      - name: Run allInOne-chart-testing (install)
+      - name: Run chart-testing (install)
         run: |
-          ct install --config ct.yaml --helm-extra-set-args "
-            --set provisionDataStore.cassandra=false
-            --set storage.type=memory
-            --set allInOne.enabled=true
-            --set agent.enabled=false
-            --set collector.enabled=false
-            --set query.enabled=false"
-  test-with-elasticsearch:
-    runs-on: ubuntu-latest
-    steps:
-        - uses: actions/checkout@v2
-          with:
-            fetch-depth: 0
-
-        - uses: ./.github/actions/prepare-k8s
+            ct install --config ct.yaml --helm-extra-set-args "--set provisionDataStore.cassandra=false --set storage.type=memory --set allInOne.enabled=true --set agent.enabled=false --set collector.enabled=false --set query.enabled=false" 
```

Comment:
this line is hard to read, please use the same formatting as in the deleted sections, one flag per line